### PR TITLE
[ci] Make CI UI table searchable

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

**Comment by tpoterba** [comment]
I tested this locally by downloading the HTML for the UI page and tinkering.

File: ci/ci/templates/index.html [hunk 1]
```diff
@@ -21,6 +21,31 @@
...
     </style>
   </head>
+  <script type="text/javascript">
+    function searchTable() {
+      var searchTerms = document.getElementById("searchBar");
+      var filter = searchTerms.value.toLowerCase();
+      var table = document.getElementById("statuses");
+      var tableRecords = table.getElementsByTagName("tr");
+
+      for (var i = 1; i < tableRecords.length; ++i) {
```
Comment (jigold) [inline ci/ci/templates/index.html]: Why does this loop start at 1?

Comment (tpoterba) [reply ci/ci/templates/index.html]: the table header is 0

**Comment by tpoterba** [comment]
good question though

**Pull request merged**